authorization/src/main/java/com/terabite/authorization/controller/AuthorizationController.java

```diff
@@ -34,4 +41,16 @@ public Payload userLoginPost() {
     public Payload userLogoutPost() {
         return new Payload("Reached logout POST");
     }
+    
+    @PutMapping("/forgot_password")
+    @ResponseStatus(HttpStatus.OK)
+    public void forgotPassword(@RequestBody String jsonEmail) {
+        forgotPasswordHelper.processForgotPassword(jsonEmail);
+    }
```

Comment:
This endpoint returns code 200 OK even if email to check is invalid. Errors should be transparent so the frontend guys know something went wrong. You should probably just send 400 Bad Request or other relevant response code + a json body with something like "status: success/failed".

Reply:
Fixed to return 400 with a bad request and 200 with a good one as well as a message. ("User found/ not found")